client_it_test.go

```diff
@@ -1043,6 +1044,62 @@ func clientStartShutdownWithNilContextTest(t *testing.T) {
 	it.Must(client.Shutdown(nil))
 }
 
+func clusterConnectionToMultipleAddrs(t *testing.T) {
+	ctx := context.Background()
+	t.Parallel()
+	tc := it.StartNewClusterWithOptions(t.Name(), it.NextPort(), 1)
+	defer tc.Shutdown()
+	mem, err := tc.RC.StartMember(ctx, tc.ClusterID)
+	if err != nil {
+		t.Fatal(err)
+	}
+	tc.MemberUUIDs = append(tc.MemberUUIDs, mem.UUID)
+	config := tc.DefaultConfig()
+	testCases := []struct {
+		name      string
+		addresses []string
+		expectErr bool
+	}{
+		{
+			name: "two valid seed addresses",
+			addresses: []string{
+				fmt.Sprintf("localhost:%d", tc.Port),
+				fmt.Sprintf("localhost:%d", mem.Port),
+			},
+		},
+		{
+			name: "first addr valid, second is invalid",
+			addresses: []string{
+				fmt.Sprintf("localhost:%d", tc.Port),
+				"non-existent.com:1234",
```

Comment:
It seems `non-existent.com` exists, could you use a domain guaranteed to be not used, maybe: `non-existent.non-existent`.

Reply:
Makes sense https://github.com/hazelcast/hazelcast-go-client/pull/885/commits/b6c392dce32ad6f9628881f5492531354940969d